docs/CreateAzureADforAzureBot.md

```diff
@@ -2,19 +2,19 @@
 
 If you would like to authenticate against your own Azure AD for development or private use of the AzureBot code, you'll need to setup your own Azure AD application.  Here are the steps you need to follow:
 
-1. Create or Utilize an Azure Active Directory (AAD) Tenant
-2. Create a new Azure AD app on the AAD tenant
-3. Configure the AAD application for use with the AzureBot
-4. Configure App Multi-Tenancy (optional)
+- [Create or Utilize an Azure Active Directory (AAD) Tenant](#createtenant)
```

Comment:
I don't believe the formatting needs to be the same as the developer setup guide.  For the AAD guide, the steps DO need to be completed in that order.  For yours if you wanted to make things more clean and similar - then I'd say you could have a pre-requisites section (could be completed in any order) that would include everything up until "Putting all together" and then you could have a numbered list of steps if you wanted.